spec/models/api_response_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe FakeData::ApiResponse do
   describe "generate_response" do
     it "returns a JSON-API object with 25 complaints" do
-      res = FakeData::ApiResponse.generate_hses_issues_response
+      res = FakeData::ApiResponse.hses_issues_response
```

Comment:
[suggestion] I think I'd keep this test on the generate method, and then add another one that tests the memoization.

Reply:
I responded to the wrong comment, but yes that is a good idea!

Reply:
[suggestion] i think you can remove lines 12-17, since you kept it on the generate method

Reply:
It's true, I decided to put it in to make sure that the method was doing what it said it would do (return issues) but it's definitely a duplicate test since there's not in terms of logic that would differ.  I can take it out!